sql/src/main/java/com/rilldata/SqlConverterEntrypoint.java

```diff
@@ -1,27 +1,82 @@
 package com.rilldata;
 
+import com.google.protobuf.InvalidProtocolBufferException;
 import com.rilldata.calcite.dialects.Dialects;
+import org.apache.calcite.sql.SqlDialect;
 import org.graalvm.nativeimage.IsolateThread;
 import org.graalvm.nativeimage.c.function.CEntryPoint;
 import org.graalvm.nativeimage.c.function.CFunctionPointer;
 import org.graalvm.nativeimage.c.function.InvokeCFunctionPointer;
 import org.graalvm.nativeimage.c.type.CCharPointer;
 import org.graalvm.nativeimage.c.type.CTypeConversion;
 import org.graalvm.word.WordFactory;
+import com.rilldata.protobuf.generated.Requests;
+import org.apache.calcite.sql.dialect.PostgresqlSqlDialect;
 
 /**
  * This class contains an entry point (a function callable from a native executable, ie C/Go executable).
  */
 public class SqlConverterEntrypoint
 {
-  private static SqlConverter sqlConverter;
+//  private static SqlConverter sqlConverter;
 
   interface AllocatorFn extends CFunctionPointer
   {
     @InvokeCFunctionPointer
     CCharPointer call(long size);
   }
 
+  public static Requests.Response transpile(Requests.Request r) {
+    Requests.TranspileRequest transpileRequest = r.getTranspileRequest();
+    String sql = transpileRequest.getSql();
+    Requests.Dialect dialect = transpileRequest.getDialect();
+    try {
+      SqlConverter sqlConverter = new SqlConverter(transpileRequest.getSchema());
+      String transpiledSql = sqlConverter.convert(sql, Dialects.valueOf(dialect.name()).getSqlDialect());
+      return Requests.Response
+          .newBuilder()
+          .setTranspileResponse(Requests.TranspileResponse.newBuilder().setSql(transpiledSql).build())
+          .build();
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Don't think we need to print the stack trace, this can happen even for a syntax error, right?